pkg/inventory/inventory.go

```diff
@@ -31,7 +32,7 @@ type Inventory interface {
 	// Load retrieves the set of object metadata from the inventory object
 	Load() (object.ObjMetadataSet, error)
 	// Store the set of object metadata in the inventory object
-	Store(objs object.ObjMetadataSet) error
+	Store(objs object.ObjMetadataSet, applyStatus map[object.ObjMetadata]status.ApplyReconcileStatus) error
```

Comment:
I'm trying to do a refactor to make the client/storage use the whole Inventory as input/output, but this map is probably fine for now. 

That said, do we want it to be an ObjectStatus list instead? 
https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/inventory/types.go#L60
Or is it important to keep it using ObjMetadata keys for now?

Reply:
We can use the ObjectStatus list here. Converting it to a map is an implementation detail inside the `Store` function.

Reply:
Updated PR. I used the `[]ObjStatus` here.